agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AiComponentInstaller.java

```diff
@@ -123,10 +123,13 @@ private static void start(Instrumentation instrumentation) {
         validateProcessorConfiguration(config);
         config.preview.authentication.validate();
         //Inject authentication configuration
-        Configuration.AadAuthentication authentication = config.preview.authentication;
-        AadAuthentication.init(authentication.type, authentication.clientId, authentication.keePassDatabasePath,
-                authentication.tenantId, authentication.clientSecret, authentication.authorityHost);
-
+        if(config.preview.authentication.enabled) {
+            Configuration.AadAuthentication authentication = config.preview.authentication;
+            AadAuthentication.init(authentication.type, authentication.clientId, authentication.keePassDatabasePath,
+                    authentication.tenantId, authentication.clientSecret, authentication.authorityHost);
+        } else {
+            AadAuthentication.init(null, null,null,null,null,null);
```

Comment:
do we have to call `init` in this case?

Reply:
had to add this because getInstance() will throw an IllegalStateException

Reply:
if authentication is disabled, AadAuthentication.init doesn't make sense.  It sounds like enabled needs to be checked before you can call AadAuthentication.getInstance() or don't throw IllegalStateException in getInstance(), but then instance can be null.  
It seems that AadAuthentication should not be a singleton.

Reply:
even I didnot like calling init with all nulls. Is there an alternative way to implement this?

Reply:
given time constraints I think it's ok to put a couple of TODO comments explaining that it's not ideal and that we want to revisit